cmd/antrea-agent/options.go

```diff
@@ -37,6 +37,7 @@ const (
 	defaultMTUGRE             = 1462
 	defaultMTUSTT             = 1500
 	defaultMTU                = 1500
+	ipsecESPHeaderSize        = 38
```

Comment:
maybe add a comment to explain where this value comes from. Also the term "header" may be a bit confusing since I think this value includes several things, one of them the ESP trailer. Maybe `ipsecESPOverhead`?

Reply:
Done.